src/main/java/io/deepstream/DeepstreamClient.java

```diff
@@ -62,15 +62,31 @@ public ConnectionState getConnectionState() {
     }
 
     public String getUid() {
-        Long timestamp = new Date().getTime();
-        Double randomNumber = Math.random() * 10000000000000000D;
-        return timestamp + "-" + randomNumber.toString().replace(".", "");
+        if( config.get( "debug" ).equals( "true" ) ) {
```

Comment:
what are thoughts on this. I'm just worried a user of the sdk will turn on debug when testing locally. However it is just to make it easier to test the generated id's in the cucumber tests